crypto/dilithium/pqcrystals_dilithium_ref_common/poly.c

```diff
@@ -354,8 +354,14 @@ void poly_uniform(poly *a,
   uint8_t buf[POLY_UNIFORM_NBLOCKS*STREAM128_BLOCKBYTES + 2];
   stream128_state state;
 
-  stream128_init(&state, seed, nonce);
-  stream128_squeezeblocks(buf, POLY_UNIFORM_NBLOCKS, &state);
+  uint8_t t[2];
+  t[0] = nonce;
+  t[1] = nonce >> 8;
+
+  SHAKE_Init(&state, SHAKE128_BLOCKSIZE);
```

Comment:
we are using a mix of `STREAM128_BLOCKBYTES` and `SHAKE128_BLOCKBYTES` throughout the code. Can we get rid of `STREAM128_BLOCKBYTES`?

Reply:
Good call out! Since all `STREAM128_BLOCKBYTES` did was point to `SHAKE128_RATE` (`#define STREAM128_BLOCKBYTES SHAKE128_RATE`) we can just use `SHAKE128_RATE` directly. I made the same change for `SHAKE256_RATE` in the removal of `symmetric.h`